### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by veronicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+1. Retrieve a user’s recently played song log
+2. Create a user interface for interacting with Spotify song logs
+3. The ability to “Up Vote” and “Down Vote” a particular song log
+4. The ability to add users to a private group
+5. User log in capabilities
+
+### Standard Requirements
+
+1. Be able to clear a user’s posted song logs
```
Comment (Talos6) [inline README.md]: do we have this one? or put this one into the stretch

File: README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# SoundBubble
+
+###### Group Members: [Martin Palanca](https://github.com/martipal), [Sherry Zhang](https://github.com/veronicz), [Yancey Yang](https://github.com/Talos6)
+
+SoundBubble is a platform that leverages Spotify's public API to build logs of a user's recently played tracks and makes this data visible to other users. Users can create and join groups to refine which users' song logs appear on their feed, as well as rate songs using an 'thumbs up' and 'thumbs down' scale.
```
Comment (martipal) [inline README.md]: Should we modify this to include commenting?

File: README.md (hunk 1 above)
Comment (martipal) [reply README.md]: yeah I don't think we have this

File: README.md (hunk 2 above)
Comment (Talos6) [reply README.md]: I think so

Comment (Talos6) [reply README.md]: but the 436I web page already shows like this

File: README.md (hunk 1 above)
Comment (veronicz) [reply README.md]: I think we interpreted this as hide/show song logs? I can reword the requirement to make that clear.

File: README.md (hunk 2 above)
Comment (veronicz) [reply README.md]: Do we want to post in Slack to let Steve know that we want to update our description? Or just update it here and include that in our demo?

Comment (Talos6) [reply README.md]: cuz this one is stretch, just update here and demo it should be fine

**Pull request merged**